apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -130,6 +137,18 @@ var NetSimRouterLogModal = module.exports = function (rootDiv, options) {
    */
   this.teacherView_ = false;
 
+  /**
+   * Pre-bound callback for use when rendering.
+   * @private {function}
+   */
+  this.boundSetRouterLogMode_ = this.setRouterLogMode_.bind(this);
```

Comment:
I think you could just do
```
this.setRouterLogMode_ = this.setRouterLogMode_.bind(this);
```
and then never need to worry about whether you want the bound or unbound one thereafter